openxlsx.Rmd

```diff
@@ -0,0 +1,581 @@
+---
+title: "Intro to Openxlsx"
+output: 
+  learnr::tutorial:
+    css: "css/style.css"
+    progressive: true
+    allow_skip: true
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
+# Author: Ciara Gribben
+# Date: Feb 2022
+# R Version: 3.6.1
+
+# Learnr Package Documentation: https://rstudio.github.io/learnr/
+
+# Packages
+library(learnr)
+library(dplyr)
+library(janitor)
+library(stringr)
+library(openxlsx)
+library(ggplot2)
+
+knitr::opts_chunk$set(echo = FALSE)
+
+# Formatting text table
+# annotation <- c("#", "##", "###", "\\*italics\\*", "\\*\\*bold\\*\\*", "\\* bulleted list")
+# formatting <- c("Header 1", "Header 2", "Header 3", "Format text to italics", "Format text to bold", "Create bulleted list")
+
+# formattext <- data.frame(Annotation = annotation, Formatting = formatting, stringsAsFactors = FALSE)
+```
+
+```{r phs-logo, fig.align='right', out.width="40%"}
+knitr::include_graphics("images/phs-logo.png")
+```
+
+## Introduction
+
+Welcome to an introduction to the openxlsx R package. This course is designed as a self-directed introduction to openxlsx for anyone in Public Health Scotland. 
+
+<div class="info_box">
+<h4>Course Info</h4>
+<ul>
+
+<li>
+
+This course is built to flow through sections and build on previous knowledge. If you're comfortable with a particular section, you can skip it.
+
+</li>
+
+<li>
+
+Some sections have multiple parts to them. Navigate the course by using the buttons at the bottom of the screen to Continue or go to the Next Topic.
+
+</li>
+
+<li>
+
+The course will also show progress through sections, a green tick will appear on sections you've completed, and it will remember your place if you decide to close your browser and come back later.
+
+</li>
+
+</ul>
+
+There are some examples designed to accompany this course, these can either be completed bit by bit as you work through the course or as a whole at the end.  There is also an exercise for you to try when you finish the course to apply what you've learned. Both can be downloaded from [Github](https://github.com/Public-Health-Scotland/).
+  </div>
+</br>
+
+
+#### What is openxlsx?
+
+Openxslx is an R package that allows us to read, write and edit xlsx files from within the RStudio interface. This will let us automate the creation and formatting of the excel files that usually accompany our publications. 
+
+The process to create these excel publication files is usually very manual and involves several copy/paste steps. This can result in errors with data being pasted in the wrong place or wrong format which can have serious consequences. For example, in 2003 the Candian power company TransAlta lost $24 million due to a copy paste error causing misaligned rows in a spreadsheet containing bids for contracts. For more examples of excel errors see this [blog post on LinkedIn](https://www.linkedin.com/pulse/7-biggest-excel-mistakes-all-time-andrew-hoag).
+
```

Comment:
Add something here about the course focussing on this specific package (with others existing), include the `install.packages()` and `library()` line at the beginning around here somewhere?